docs/developer_workflow.md

```diff
@@ -88,6 +88,20 @@ This will configure [kubectl](https://kubernetes.io/docs/reference/kubectl/) for
     make test-kubectl-workload TEST_CLUSTER_NAME=<>
     ```
 
+## Create a test workload cluster with topology
+
+1. Create a workload cluster:
+
+    ```shell
+    make test-cc-cluster-create
+    ```
+
+   Optionally, to use a unique cluster name:
+
+    ```shell
+    make test-cc-cluster-create TEST_CLUSTER_NAME=<>
```

Comment:
Thanks for the docs! If documenting `TEST_CLUSTER_NAME` lets remove the new var `TEST_TOPOLOGY_CLUSTER_NAME` in the Makefile and reuse `TEST_CLUSTER_NAME`